packages/terra-status-view/src/StatusView.scss

```diff
@@ -0,0 +1,111 @@
+  }
+
+  .glyph > .error {
+    background: var(--terra-status-view-glyph-error, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='#E50000' d='M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z'/><path fill='#FFF' d='M7 21.5h34v5H7v-5z'/></svg>"));
+  }
+
+  .glyph > .no-service {
+    background: var(--terra-status-view-glyph-no-service, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48' class='is-bidi'><path d='M19 17.9c-4.1 0-7.8 1.7-10.5 4.3l-3.1-3.1c3.5-3.5 8.3-5.6 13.6-5.6s10.1 2.1 13.6 5.6L29.7 22c-.1 0-.1.1-.2.1-2.8-2.6-6.4-4.2-10.5-4.2zm15.7-.9l3.3-3.3C33.1 9 26.4 6 19 6S4.9 9 0 13.7L3.3 17c4-4 9.6-6.5 15.7-6.5S30.7 13 34.7 17zM21.2 28.9c-.7-.4-1.4-.6-2.2-.6-1.2 0-2.3.5-3.1 1.3l3.1 3.1 1.1-1.1c.3-.9.7-1.8 1.1-2.7zm1.8-2.4c1-1.1 2.2-2.1 3.5-2.9-2-1.6-4.6-2.6-7.4-2.6-3.2 0-6.2 1.3-8.4 3.4l3.1 3.1c1.3-1.3 3.2-2.2 5.2-2.2 1.5 0 2.8.4 4 1.2z'/><path d='M34 24c-6.6 0-12 5.4-12 12s5.4 12 12 12 12-5.4 12-12-5.4-12-12-12zm-9 12c0-5 4-9 9-9 1.9 0 3.7.6 5.2 1.7L26.7 41.2C25.6 39.7 25 37.9 25 36zm9 9c-1.9 0-3.7-.6-5.2-1.7l12.5-12.5c1 1.5 1.7 3.3 1.7 5.2 0 5-4 9-9 9z'/></svg>"));
+  }
+
+  .glyph > .custom {
+    background: var(--terra-status-view-glyph-custom, none);
+  }
+
+  .message-content-group {
+    align-items: center;
+    display: block;
+    flex-direction: column;
+    padding-left: 10px;
+    padding-right: 10px;
+    width: 100%;
+  }
+
+  .title {
+    color: var(--terra-status-view-title-color, #000);
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    font-family: var(--terra-status-view-title-font-family, 'Roboto');
+    font-size: var(--terra-status-view-title-font-size, 24px);
+    text-align: center;
+    width: 100%;
+  }
+
+  .message {
+    color: var(--terra-status-view-message-color, #000);
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    font-family: var(--terra-status-view-message-font-family, 'Roboto');
+    font-size: var(--terra-status-view-message-font-size, 14px);
+    padding-top: var(--terra-status-view-message-padding-top, 10px);
+    text-align: var(--terra-status-view-message-text-align, center);
+    width: 100%;
+  }
+
+  .divider {
+    display: var(--terra-status-view-divider-display, block);
+    padding-top: var(--terra-status-view-divider-padding-top, 20px);
+    width: 100%;
+  }
+
+  .buttons {
+    display: flex;
+    flex: 0 0 auto;
+    flex-direction: var(--terra-status-view-buttons-flex-direction-small-width, row);
+    justify-content: center;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-top: var(--terra-status-view-buttons-padding-top, 10px);
```

Comment:
should this be `--terra-status-view-footer-*` instead?

Reply:
this is for the buttons div, which may or may not display depending on if children are passed in; therefore, calling this footer wouldn't be consistent as the descriptive message would be the last item

Reply:
But if it doesn’t display then isn’t there just not a footer? Also if footer isn’t he right word perhaps actions then?

Reply:
right, by not displaying this then the view will shrink vertically and the message div will be the bottom, if no message then will be the title div, the outer div (.status-view) has padding-bottom applied to it to act as the bottom margin no matter which element is displays last

actions is fine for the name:  --terra-status-view-actions-padding-top, I am good with that

Reply:
+1 for actions